text/src/csplit.rs

```diff
@@ -130,10 +150,10 @@ impl OutputState {
             .write(true)
             .create(true)
             .truncate(true)
-            .open(&out_fn)?;
+            .open(out_fn.clone())?;
```

Comment:
Why are we cloning here? The original solution with providing a reference should've worked